float64 stamp
---
bool  	result
float64   information
